dnf5-5.1.11 FTBFS in openSUSE due to msgmerge failure

I tried to build dnf5-5.1.11, but it fails due to a msgmerge failure:

```
[   24s] [  1%] Generating sv.gmo
[   24s] cd /home/abuild/rpmbuild/BUILD/dnf5-5.1.11/build/libdnf5-cli/po && /usr/bin/msgmerge --quiet --update --backup=none -s /home/abuild/rpmbuild/BUILD/dnf5-5.1.11/build/libdnf5-cli/po/sv.po /home/abuild/rpmbuild/BUILD/dnf5-5.1.11/libdnf5-cli/po/libdnf5-cli.pot
[   24s] /home/abuild/rpmbuild/BUILD/dnf5-5.1.11/build/libdnf5-cli/po/sv.po: warning: Charset "CHARSET" is not a portable encoding name.
[   24s]                                                                              Message conversion to user's charset might not work.
[   24s] /home/abuild/rpmbuild/BUILD/dnf5-5.1.11/build/libdnf5-cli/po/sv.po:124: duplicate message definition...
[   24s] /home/abuild/rpmbuild/BUILD/dnf5-5.1.11/build/libdnf5-cli/po/sv.po:3: ...this is the location of the first definition
[   24s] /usr/bin/msgmerge: found 1 fatal error
[   24s] make[2]: *** [libdnf5-cli/po/CMakeFiles/translations_2.dir/build.make:84: libdnf5-cli/po/sv.gmo] Error 1
[   24s] make[2]: Leaving directory '/home/abuild/rpmbuild/BUILD/dnf5-5.1.11/build'
[   24s] make[1]: *** [CMakeFiles/Makefile2:1608: libdnf5-cli/po/CMakeFiles/translations_2.dir/all] Error 2
```

--- a/libdnf5-cli/pt_BR.po
+++ b/libdnf5-cli/pt_BR.po
@@ -116,26 +116,6 @@ msgstr "Operação abortada pelo usuário."
 msgid "Failed to resolve the transaction"
 msgstr "Falha ao resolver a transação"
 
-# SOME DESCRIPTIVE TITLE.
-# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
-# This file is distributed under the same license as the PACKAGE package.
-# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
-#
-#: exception.cpp:49
-#, fuzzy
-msgid ""
-msgstr ""
-"Project-Id-Version: PACKAGE VERSION\n"
-"Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2024-01-09 02:52+0000\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
-"Language: \n"
-"MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=CHARSET\n"
-"Content-Transfer-Encoding: 8bit\n"
-
 #: session.cpp:79
 msgid "Missing command"
 msgstr "Faltando comando"

--- a/libdnf5-cli/sv.po
+++ b/libdnf5-cli/sv.po
@@ -117,26 +117,6 @@ msgstr "Operation avbröts av användaren."
 msgid "Failed to resolve the transaction"
 msgstr "Misslyckades att lösa transaktionen"
 
-# SOME DESCRIPTIVE TITLE.
-# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
-# This file is distributed under the same license as the PACKAGE package.
-# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
-#
-#: exception.cpp:49
-#, fuzzy
-msgid ""
-msgstr ""
-"Project-Id-Version: PACKAGE VERSION\n"
-"Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2024-01-09 02:52+0000\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
-"Language: \n"
-"MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=CHARSET\n"
-"Content-Transfer-Encoding: 8bit\n"
-
 #: session.cpp:79
 msgid "Missing command"
 msgstr "Saknar kommando"

